packages/core/src/App/Containers/RealAccountSignup/helpers/constants.js

```diff
@@ -34,7 +34,7 @@ export const getAccountTitle = (short_code, { is_im_residence = false } = {}) =>
 
     switch (short_code) {
         case 'svg':
-            return localize('Real');
+            return localize('Deriv account');
```

Comment:
Are we changing references to `Real` to `Deriv account`?

Reply:
Thanks for pointing it out, I'm not sure myself. Let me get a confirmation on this one.